### PR TITLE
sum-of-multiples: update tests to v1.3.0

### LINKED ISSUE #1567: sum-of-multiples: Example does not match canonical data
If we enter 0 as a multiple in the current example of `sum-of-multiples`, a ZeroDivisionError is raised.
[Canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/sum-of-multiples/canonical-data.json) require processing the case when zero is in the list of multiples.
The example file does not take that into account, neither does the tests file.
I came across this issue when fixing the inconsistency of `sum-of-multiples-test.py` with canonical data.

### PR DESCRIPTION
**Pull request by patiencedaur**
Fix #1567

### CODE REVIEW

**Comment by cmccandless** [comment]
You've got a commit from another branch in this one. Please remove it.

File: exercises/sum-of-multiples/example.py [hunk 1]
```diff
@@ -1,4 +1,7 @@
 def sum_of_multiples(limit, multiples):
-    return sum(value for value in range(limit)
+    if 0 in multiples:
+        return 0
+    else:
+        return sum(value for value in range(limit)
                if any(value % multiple == 0
```
Comment (cmccandless) [inline exercises/sum-of-multiples/example.py]: Needs more indent on line 6.

```suggestion
                   if any(value % multiple == 0
```

**Comment by DrJPizzle** [comment]
This is not the correct behaviour. An extra zero should not affect the answer.
PR to follow.

**Comment by DrJPizzle** [comment]
See issue #1567 
and PR #1592

**Comment by cmccandless** [comment]
This PR is abandoned; closing.

**Pull request closed without merging**